awswrangler/dynamodb/_read.py

```diff
@@ -313,6 +313,7 @@ def _read_items(
 
```

Comment:
Does  it make sense to add a debug call in the `_read_scan` function about which segment is being processed?